packages/react-native-win32/src/Libraries/Components/Touchable/TouchableWin32.tsx

```diff
@@ -257,6 +263,10 @@ export class TouchableWin32 extends React.Component<ITouchableWin32Props, IInter
     );
   }
 
+  public focus = () => {
```

Comment:
@PPatBoyd, this is one of the approaches I was talking about offline, effectively for class components we can wrap everything in this way. Outer components hold RefObject<TouchableWin32>s that can imperatively call focus on. I think there's a similar thing we can do with function components using imperativeHandles.